addon/components/simple-typeahead/template.hbs

```diff
@@ -1,16 +1,25 @@
 <div class="input">
-  <input
+  {{input
```

Comment:
see my comment on PR #3 .  tl;dr keydown event handler doesn't work w/ handlebars version, and when possible, probably better to stick w/ native DOM.